### PR TITLE
PIC-2801

### PR DESCRIPTION
**Pull request by vb-moj**
. Added initial namespace for court probation e2e.

### CODE REVIEW

File: namespaces/live.cloud-platform.service.justice.gov.uk/court-probation-e2e/resources/variables.tf [hunk 1]
```diff
@@ -0,0 +1,32 @@
+variable "vpc_name" {
+}
+
+variable "application" {
+  default = "HMPPS  Court Probation"
```
Comment (johnyfarrar) [inline namespaces/live.cloud-platform.service.justice.gov.uk/court-probation-e2e/resources/variables.tf:5]: Are the additional spaces a typo or a requirement?

Comment (vb-moj) [reply namespaces/live.cloud-platform.service.justice.gov.uk/court-probation-e2e/resources/variables.tf:5]: Looks like a typo. will remove.Ta

File: namespaces/live.cloud-platform.service.justice.gov.uk/court-probation-e2e/00-namespace.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: court-probation-e2e
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "false"
+    cloud-platform.justice.gov.uk/environment-name: "dev"
+    cloud-platform.justice.gov.uk/namespace: "court-probation-e2e"
```
Comment (rschallapalli) [inline namespaces/live.cloud-platform.service.justice.gov.uk/court-probation-e2e/00-namespace.yaml:8]: Is it much easier/less-complex to use the existing court-probation-* namespace? @jhevans

Comment (vb-moj) [reply namespaces/live.cloud-platform.service.justice.gov.uk/court-probation-e2e/00-namespace.yaml:8]: It is easier to maintain with the new namespace. But I go with the consensus.

Comment (vb-moj) [reply namespaces/live.cloud-platform.service.justice.gov.uk/court-probation-e2e/00-namespace.yaml:8]: From @jhevans

**Comment by vb-moj** [comment]
No need for this PR as we will use the existing court-probation-dev namespace.

**Pull request closed without merging**